### PR TITLE
qp 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by netoht**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by netoht** [comment]
Please administrators, verify the `continuous-integration/jenkins/ghprb` and rerun the build. It has broken because the Jenkins didn't get access to internet in that moment.

**Comment by Bo98** [comment]
Yes, we had an outage over the weekend and have only recovered now.

I'll restart the build but it may be a little slow as there's ~130 pull requests in the queue.

@BrewTestBot test this please

File: Formula/qp.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  desc "The query-pipe: command-line (ND)JSON querying"
+  homepage "https://github.com/paybase/qp"
+  url "https://github.com/paybase/qp/archive/1.0.1.tar.gz"
+  sha256 "6ef12fd4494262899ee12cc1ac0361ec0dd7b67e29c6ac6899d1df21efc7642b"
+
+  depends_on "quickjs" => :build
+
+  def install
+    mkdir bin.to_s
+    system "curl", "https://unpkg.com/@paybase/csp@1.0.8/dist/esm/index.js", "--output", "./src/csp.js"
```
Comment (SMillerDev) [inline Formula/qp.rb]: This is not acceptable, please use a resource block.

Comment (netoht) [reply Formula/qp.rb]: Thank you for your feedback, please check it again. I changed to use resource block.

**Comment by SMillerDev** [comment]
Can you please squash your commits?

**Comment by netoht** [comment]
> Can you please squash your commits?

Of course, I have done!

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
@netoht Thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**